staging/src/k8s.io/apiserver/pkg/endpoints/handlers/patch.go

```diff
@@ -122,8 +129,8 @@ func PatchResource(r rest.Patcher, scope RequestScope, admit admission.Interface
 			kind:            scope.Kind,
 			resource:        scope.Resource,
 
-			createValidation: rest.AdmissionToValidateObjectFunc(admit, staticAdmissionAttributes),
-			updateValidation: rest.AdmissionToValidateObjectUpdateFunc(admit, staticAdmissionAttributes),
+			createValidation: rest.AdmissionToValidateObjectFunc(admit, admission.NewAttributesRecord(nil, nil, scope.Kind, namespace, name, scope.Resource, scope.Subresource, admission.Create, userInfo)),
```

Comment:
If I try to create a pod on update, which is [not allowed](https://github.com/kubernetes/kubernetes/blob/90f681be107cf67aa610b836a6a45f99e0bf04f2/staging/src/k8s.io/apiserver/pkg/registry/generic/registry/store.go#L568). Will this still have effect?  For example, will this increase quota usage?

Reply:
@CaoShuFeng Thanks for reviewing. I do find some codes need to be trimed in resourcequota controller as u see in my following commit. And AFAICT resourcequotas would still be fine after these changes WDYT?

Reply:
Sorry that I have no suggestions about how to fix this issue. But I'd be happy to point it out when I see something doesn't work as expected.

Maybe we can ask for good suggestions from more experienced developers. :)

Reply:
Actually, I don't see an invocation of `NewObjectCountEvaluator` with `allowCreateOnUpdate=true`, not in kube and not in OpenShift. Maybe @derekwaynecarr can shed some light on that.

Reply:
In other words: did we miss creations on update in the quota mechanism before this PR?

Reply:
@sttts I don't see either even search through the whole project. It should have been pruned before 😅